### PR TITLE
Nodemailer

### PR DESCRIPTION
**Pull request by Jood80**
### In thisss PR :octocat: : 

- :heavy_check_mark:  The customer was able to get a confirmation message with his/her reservation date #81 
- :heavy_check_mark:  The used service is [nodemailer](https://nodemailer.com)
---
### How it's done:
- in ` helper/nodeMailer.js` you can see the work that's been done. basically our concern here is with `transporter` and  `mailOptions` that make the configuration of nodemailer. 
- on `createTransport`, the admin email and his password should be provided, so I created a Gmail account for that usage, then paste them on our `createTransport`,
- after so this has been provided too with a transport layer security _TLS_ that uses cipher algorithm  'SSLv3'.
- after running the code for the first time l got this email as an Admin
image
- just press `Check activity` and to get this 
image
- then `yes, it was me` **obviously** :smirk:
image

- Right now you have to go into your `Account>settings> security` and put "less secure app" **ON**
image
---
### Testing
****Note**: the `component/calendar` branch must be merged to master before testing this dude.**

Write these following commands down in your terminal :musical_keyboard: 
-    pull origin nodemailer
- write down these two lines in your `config.env` file
>  GMAIL_USER=XXXXXX@gmail.com
> GMAIL_PASS=XXXX
-  npm run dev
-  Go to http://localhost:3000/reservation on your browser.
-  Fill out each required field- including time&date from the calendar tab-  and submit the form.
-  Check the email that you've used in your registration form :drum: and...

---
### Wallaaa!! you should receive by now some weird email that looks like this in your inbox

image

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const app = require('./app.js')
+const app = require('./app.js');
 
-const PORT = process.env.PORT || 5000
+const PORT = process.env.PORT || 5000;
 
 app.listen(PORT, () => {
-  console.log(`The things, the things are happening on ${PORT}!!`)
```
Comment (okjake) [inline server/index.js:6]: Boo, I liked the things happening 😄

**Review by okjake — APPROVED** [review]
Looks good, need to format the actual mail of course but this is a good start.

For production I'd look into transactional mail services like Sendgrid, Mailgun - it can cause problems using Gmail especially if you end up sending a lot of mails, but for prototyping it's great.

**Comment by Jood80** [comment]
> Looks good, need to format the actual mail of course but this is a good start.
> 
> For production I'd look into transactional mail services like Sendgrid, Mailgun - it can cause problems using Gmail especially if you end up sending a lot of mails, but for prototyping it's great.

@okjake Cool!! well it seems that a lot of interesting stuff is coming up in this project even in the refactoring stage :bowtie: :sparkler:  
Can't waiiiittt to try themm :fire: :metal:

**Review by sara219 — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
🤩🤩🤩🤩🤩🤩

**Review by tasneembhiri — APPROVED** [review]
:octocat:

**Pull request merged**